server.js

```diff
@@ -23,6 +41,37 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thoughts)
+})
```

Comment:
according to documentation `get "/"` need to return 20 thoughts,

Reply:
I dont really understand this point? It is returning max 20 thoughts?

Reply:
You used ` get /thoughts` and the documentation forced you to use ` get /`